lib/systemFlake.nix

```diff
@@ -141,6 +156,10 @@ let
                 networking.hostName = hostname;
               })
 
+              (optionalAttrs (options ? networking.domain) {
+                networking.domain = domain;
+              })
+
```

Comment:
```suggestion
                networking.hostName = lib.mkDefault hostname;
              })

              (optionalAttrs (options ? networking.domain) {
                networking.domain = lib.mkDefault domain;
              })

```
In case you don't want to pass domains to fup.

Reply:
Good point! However, I guess the correct implementation I guess is slightly different, I'll push in a few seconds.

Reply:
got it, yeah that looks right